features.py

```diff
@@ -0,0 +1,42 @@
+from feature_block import *
+from feature_set import *
+
+import argparse
+
+import halfkp
+
+_feature_modules = [halfkp]
```

Comment:
Is this where you would add new architectures?  If so, a description here would be great.